config/settings.py

```diff
@@ -89,6 +90,16 @@ def pipe_delim(pipe_string):
 }
 DATABASES[decouple_config('DJANGO_DATABASE_KEY')] = decouple_config('DJANGO_DATABASE_URL', cast=db_url)
 
+LIVE_DEMO = decouple_config('LIVE_DEMO', default=False, cast=bool)
+TEST = decouple_config('TEST', default=False, cast=bool)
```

Comment:
Is this a default Django setting? If not, we should namespace it too.

Reply:
This appears to be set up by `pytest-django` automatically when we run the tests.